crates/ty_python_semantic/src/types/diagnostic.rs

```diff
@@ -666,7 +813,26 @@ declare_lint! {
 }
 
 declare_lint! {
-    /// TODO #14889
+    /// ## What it does
+    /// Checks for type variables that has only one constraint.
```

Comment:
```suggestion
    /// Checks for type variables that have only one constraint.
```

Can we also add a link to https://docs.python.org/3/library/typing.html#typing.TypeVar, which has a nice explanation of the difference between bound and constrained typevars